weather_dl_v2/cli/README.md

```diff
@@ -0,0 +1,34 @@
+# dl-cli
```

Comment:
In general can pick content from: https://github.com/google/weather-tools/blob/dl-v2/weather_dl_v2/nginx-server/README.md

Reply:
updated in [e289aa3](https://github.com/google/weather-tools/pull/346/commits/e289aa30dc12831fce0ad664762bf7634b24f378)